src/components/Card/Card.js

```diff
@@ -15,6 +21,8 @@ function Card({product, favourites, handleFavoriteButtonClick}) {
                 product={product}
                 active={favourites.some((item) => item.id === product.id)}
```

Comment:
active is not perfect as naming, because IconButtons renders two icons, maybe change to `favoriteActive`